apps/src/applab/Actions.js

```diff
@@ -0,0 +1,42 @@
+/** @file Redux action-creators for App Lab.
```

Comment:
Might be too late, but my preference would be that this is lower case actions since we're not exposing a constructor/class.